packages/cli/bin/designsystemet.ts

```diff
@@ -56,11 +65,13 @@ function makeTokenCommands() {
         },
       };
 
+      if (dry) {
+        console.log(`Running dry run for `);
```

Comment:
> Running dry run for

for what? 😄

Kanskje både denne og den andre beskjeden kan vere: "Performing dry run, no files will be written" elns